app/views/companies/_navbar.html.erb

```diff
@@ -0,0 +1,41 @@
+<nav class="navbar navbar-expand-md navbar-dark fixed-top bg-dark">
+   <div class="container-fluid">
+
+      <a class="navbar-brand" href="<%= root_path %>">JSCLook</a>
+
+      <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarCollapse"
+         aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation">
+         <span class="navbar-toggler-icon"></span>
+      </button>
+
+      <div class="collapse navbar-collapse" id="navbarCollapse">
+         <ul class="navbar-nav me-auto mb-2 mb-md-0">
+            <li class="nav-item active">
+               <a class="nav-link" href="<%= root_path %>">Home </a>
+            </li>
+            <li class="nav-item">
+               <a class="nav-link" href="https://www.gpw.pl/">GPW</a>
+            </li>
+            <li class="nav-item">
+               <a class="nav-link" href="https://newconnect.pl/">NewConnect</a>
+            </li>
+         </ul>
+
+         <!--
+         <div  class="mx-auto">
+            <form class="d-flex">
+               <input class="form-control mr-sm-2" type="text" placeholder="Search" aria-label="Search">
+               <button class="btn btn-outline-secondary btn-sm" type="submit" role="button">Search</button>
+            </form>
+         </div> -->
+
+         <div class="d-grid gap-2 d-md-flex justify-content-md-end">
+            <ul class="navbar-nav">
+               <li class="nav-item">
+                  <button role="button" class="btn btn-outline-success">Login</button>
+               </li>
+            </ul>
+         </div>
+      </div>
+   </div>
+</nav>
```

Comment:
One more thing:

I think this file `app/views/companies/_navbar.html.erb` does not belong to `companies` namespace. I would try to move it to `app/views/layouts/_navbar.html.erb`